### PR TITLE
docs: update verification report template

### PR DESCRIPTION
**Pull request by mathiasbio**
### This PR:

Adds a section in the verification report template for verifying integrity of storage and upload of files, if these steps are affected in the new release. According to recommendation made here: https://github.com/Clinical-Genomics/BALSAMIC/issues/1171

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Review by ivadym — COMMENTED** [review]
Awesome! Excellent job addressing the recommendation from the previous verification ⭐

File: .github/ISSUE_TEMPLATE/verification_report.md [hunk 1]
```diff
@@ -86,6 +86,33 @@ description, and status (Pass/Fail).
...
+Table of potentially affected steps in Storage and Upload of files:
+
+| Processes                                | Affected in current version | Affected workflows     | 
+|------------------------------------------|-----------------------------|------------------------|
+| New files to be delivered                | Yes/No                      | ALL / TGA / WGS / UMI  | 
+| Changes of file-names of delivered files | Yes/No                      | ALL / TGA / WGS / UMI  | 
+| New files to be stored                   | Yes/No                      | ALL / TGA / WGS / UMI  | 
+| Changes of file-names of stored files    | Yes/No                      | ALL / TGA / WGS / UMI  | 
+| Changes to Housekeeper IDs               | Yes/No                      | ALL / TGA / WGS / UMI  | 
+| Changes to Scout yaml creation           | Yes/No                      | ALL / TGA / WGS / UMI  |
```
[the remark this answers is not in the document]
Comment (mathiasbio) [reply .github/ISSUE_TEMPLATE/verification_report.md]: Nice : )

**Comment by mathiasbio** [comment]
> Awesome! Excellent job addressing the recommendation from the previous verification ⭐

Thanks for the review! Just a double-sanity-check here, should I actually merge this into Master?

**Review by ivadym — APPROVED** [review]
💯 👍

**Comment by ivadym** [comment]
> Thanks for the review! Just a double-sanity-check here, should I actually merge this into Master?

Yes, it's just an issue template! No need to change the CHANGELOG either. Just make sure to merge master into your branch :)

**Pull request merged**